website/docs/docs/core/connect-data-platform/snowflake-setup.md

```diff
@@ -124,6 +124,8 @@ Along with adding the `authenticator` parameter, be sure to run `alter account s
 
 To use key pair authentication, omit a `password` and instead provide a `private_key_path` and, optionally, a `private_key_passphrase` in your target. **Note:** Versions of dbt before 0.16.0 required that private keys were encrypted and a `private_key_passphrase` was provided. This behavior was changed in dbt v0.16.0.
 
+From dbt v1.5.0 onwards, it is possible to provide a `private_key` (Base64-encoded DER-encoded key bytes) string instead of a `private_key_path`.
```

Comment:
hey @b-per , tweaked it slightly and also added a base64 link - does this read correctly, technically?

```suggestion
Starting from [dbt v1.5.0](/docs/dbt-versions/core), you have the option to use a `private_key` string instead of a `private_key_path`. The `private_key` string should be in [Base64-encoded DER format](https://en.wikipedia.org/wiki/Base64#:~:text=In%20computer%20programming%2C%20Base64%20is,four%206%2Dbit%20Base64%20digits.), representing the key bytes. 

```

Reply:
Thanks for the update on the version.

I am not too sure about the link to Wikipedia for base64 though. This is a quick explanation of base64 but actually not the full format needed for the key in that case. This might be OK to link but in that case, I would also [link the page of the Snowflake docs](https://docs.snowflake.com/developer-guide/python-connector/python-connector-example#using-key-pair-authentication-key-pair-rotation) that show how this key is generated.

Reply:
sounds great! thank you for providing that link - ok made this change now and happy to approve and merge this for you ✨